package/gluon-core/i18n/gluon-core.pot

```diff
@@ -0,0 +1,38 @@
+msgid ""
+msgstr "Content-Type: text/plain; charset=UTF-8"
+
+msgid "Autoupdater branch"
+msgstr ""
+
+msgid "Domain"
+msgstr ""
+
+msgid "Firmware release"
+msgstr ""
+
+msgid "Gluon version"
+msgstr ""
+
+msgid "Hardware model"
+msgstr ""
+
+msgid "Hostname"
+msgstr ""
+
+msgid "MAC address"
+msgstr ""
+
+msgid "Public VPN key"
+msgstr ""
+
+msgid "Site"
+msgstr ""
+
+msgid "Site version"
+msgstr ""
+
+msgid "Switch type"
+msgstr ""
+
+msgid "disabled"
```

Comment:
Hmm, ich sehe gerade, die Strings werden für den Config Mode in `gluon-web-admin/i18n` übersetzt; auf der Konsole haben wir gar keinen Support für i18n. Wobei es etwas seltsam ist, dass Strings aus `gluon-core` ihre Übersetzungen in `gluon-web-admin` haben... ich schaue mir das an.

Reply:
#3503